sdk/Directory.Build.props

```diff
@@ -3,10 +3,10 @@
     <IsMgmtProject Condition="'$(IsMgmtProject)' == '' and $(MSBuildProjectName.Contains('.Management.'))">true</IsMgmtProject>
     <IsDataPlaneProject Condition="'$(IsDataPlaneProject)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Azure.')) and '$(IsMgmtProject)' != 'true'">true</IsDataPlaneProject>
   </PropertyGroup>  
-  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
+  <ItemGroup Condition="'$(IsTestProject)' == 'true' or '$(IsPerfProject)' == 'true'">
```

Comment:
We may want to do this for Stress Projects too, as they're likely to have the same general infrastructure.

Reply:
Done.